docs/workspaces.md

```diff
@@ -366,6 +366,9 @@ to define when a `Task` should be executed. For more information, see the [`runA
 When a `PersistentVolumeClaim` is used as volume source for a `Workspace` in a `PipelineRun`,
 an Affinity Assistant will be created. For more information, see the [`Affinity Assistants` documentation](affinityassistants.md).
 
+**Note**: It is not allowed to bind multiple [`PersistentVolumeClaim` based workspaces](#using-persistentvolumeclaims-as-volumesource) to a `PipelineTaskRun` in the `coschedule workspaces` coschedule mode due to potential Availability Zone conflict.
```

Comment:
I'm not sure if people will understand "PipelineTaskRun"; maybe you can say "when `coschedule` is set to "workspaces" or `disable-affinity-assistant` is set to false, you cannot bind multiple PVC-backed workspaces to the same taskrun in a pipelinerun due to potential Availability Zone conflict.

Reply:
Thanks, I have remove the term `PipelineTaskRun` in the whole doc.